packetbeat/flows/worker_test.go

```diff
@@ -126,4 +126,27 @@ func TestCreateEvent(t *testing.T) {
 			t.Fatal(err)
 		}
 	}
+
+	// when enableDeltaFlowReporting is true, the flow stats should be reset
+	expectbiFlow := &biFlow{
+		id:       id.rawFlowID,
+		killed:   1,
+		createTS: start,
+		ts:       end,
+		dir:      flowDirForward,
+	}
+	expectbiFlow.stats[0] = &flowStats{uintFlags: []uint8{1, 1}, uints: []uint64{0, 0}}
+	expectbiFlow.stats[1] = &flowStats{uintFlags: []uint8{1, 1}, uints: []uint64{0, 0}}
```

Comment:
I'd feel a bit more comfortable asserting that `bif.stats[0].uints` is not equal `expectbiFlow.stats[0].uints` here, in case the test above changes, I'll leave it at your discretion.

Reply:
done